scripts/check_determinism.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Run this script from the repo root with e.g.:
+#
+#   ./scripts/check_determinism.sh rutland
+#
+# passing the region to be checked as the first positional arg.
+#
+# If files differ, the protobuf are converted to sorted JSON to get diff.
+
+# Use given region
```

Comment:
Decision to hardcode the year and country looks good to me. No need to complicate the script until we spot some potential problem in another configuration

Reply:
Yes there are various options we could add (e.g. pass a year as well as a region, pass seeds in optionally) but agree these could be added another time.